var/spack/repos/builtin/packages/structure/package.py

```diff
@@ -15,12 +15,14 @@ class Structure(MakefilePackage):
 
     version('2.3.4', sha256='f2b72b9189a514f53e921bbdc1aa3dbaca7ac34a8467af1f972c7e4fc9c0bb37')
 
-    depends_on('jdk', type=('build', 'run'))
-
```

Comment:
According to this: https://web.stanford.edu/group/pritchardlab/structure_software/release_versions/v2.3.4/html/install.html `jdk` is needed to install the front-end?

Reply:
The recipe as is doesn't build the front end (which is a different source download it seems), so I think we can safely remove it.